services/render.tsx

```diff
@@ -47,17 +51,25 @@ function EmbeddedLink(node: any, isInline = false): ReactElement {
   const captionClasses = [...inlineClasses, !isInline && 'pb-4'];
   return (
     <figure className={classNames(figureClasses)}>
-      <EmbedContent content={node.meta} classes={classNames(inlineClasses)} />
+      <EmbedContent
+        textDirection={textDirection}
+        content={node.meta}
+        classes={classNames(inlineClasses)}
+      />
       {node.caption && (
-        <figcaption className={classNames(captionClasses)}>
-          <em>{node.caption}</em>
+        <figcaption dir={textDirection} className={classNames(captionClasses)}>
+          <em>{node.caption} </em>
```

Comment:
Do we need the empty space here or can we remove it?

Reply:
not neccesary! removed it